kratos/utilities/function_parser_utility.cpp

```diff
@@ -182,11 +182,24 @@ void BasicGenericFunctionUtility::InitializeParser()
         /* Store variable names and pointers. */
         const te_variable vars[] = {{"x", &x}, {"y", &y}, {"z", &z}, {"t", &t}, {"X", &X}, {"Y", &Y}, {"Z", &Z}};
 
+        auto fct_checker = [](const bool IsValid, const std::string& rFunction, const int ErrPos){
+            if (IsValid) return;
+
+            std::stringstream ss;
+            ss << "\nParsing error in function: " << rFunction << '\n';
+            ss <<   "Error occurred near here : ";
+            for(int i=0; i<ErrPos-1; ++i) ss << ' ';
+            ss << "^ (char ["<< ErrPos-1 << "])\n";
+            ss << "Check your locale (e.g. if \".\" or \",\" is used as decimal point)";
```

Comment:
```suggestion
            ss << "Ensure the expression is valid and check your locale (e.g. if \".\" or \",\" is used as decimal point)";
```

I know it should be obvious I't probably better for it to be here.